src/lib/components/Toggle/Toggle.js

```diff
@@ -66,4 +66,4 @@ ToggleComponent.defaultProps = {
   overridableId: "",
 };
 
-export default Overridable.component("SearchFilters.Toggle", ToggleComponent);
+export default Overridable.component("SearchFilters.Toggle.element", ToggleComponent);
```

Comment:
wy is this change needed? it is not consistent with the rest of the components ids and potentially a breaking change